src/utils/validateLocusCondition.ts

```diff
@@ -0,0 +1,50 @@
+import { FuncStepTypesEnum } from '@core/enum/func-step-types-enum'
+import dtreeStore from '@store/dtree'
+import filterStore from '@store/filter'
+
+export const validateLocusCondition = (
+  value: string,
+  setIsErrorVisible: (arg: boolean) => void,
+  groupName?: string,
+  currentStepIndex?: number,
```

Comment:
I would rather define an interface for the parameters of this function and pass them in an object. Because when you have multiple params (more than 2, for example) it's quite easy to mess up with the order. Passing an object is more convenient in this case.